digitalpy/core/component_management/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""This core package contains functions related to Managment of components, this includes discovery, 
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D400 First line should end with a period